stub-idp/src/main/java/uk/gov/ida/stub/idp/resources/idp/LogoutPageResource.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.ida.stub.idp.resources.idp;
+
+import uk.gov.ida.stub.idp.Urls;
+import uk.gov.ida.stub.idp.cookies.CookieFactory;
+import uk.gov.ida.stub.idp.cookies.CookieNames;
+import uk.gov.ida.stub.idp.cookies.HttpOnlyNewCookie;
+
+import javax.inject.Inject;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+import java.util.Date;
+
+@Path(Urls.LOGOUT_RESOURCE)
+@Produces(MediaType.TEXT_HTML)
+public class LogoutPageResource {
+
+    @Inject
+    public LogoutPageResource() {
+
+    }
+
+    @GET
+    public Response get(@PathParam(Urls.IDP_ID_PARAM) @NotNull String idpName) {
+        return Response.seeOther(UriBuilder.fromPath("").host("google.co.uk").build())
```

Comment:
perhaps use a better URL to redirect people to - maybe the single idp page that is seen when people are not logged in